project_proposal/tex/mainpage.tex

```diff
@@ -192,6 +192,10 @@ \section{Project Organisation}
 etc. Kenneth Munk Christiansen was chosen to be Scrum Master, so he ensures that the Scrum
 framework is understood and is used correctly throughout the project. Like the
 rest of the group, they are also a part of the Development Team.
+For easy version control and collaboration github will be used. Github is a version 
+control system that allows each teammember to download the code, modify it and upload the new version.
```

Comment:
GitHub is the hosting service that we use, to have a centralize repository for our project.
Git is a revision control system.

Should we have it clear like that or does that not matter?